burn-book/src/building-blocks/metric.md

```diff
@@ -0,0 +1,117 @@
+# Metric
+
+When working with the learner, you have the option to record metrics that will be monitored
+throughout the training process. We currently offer a restricted range of metrics.
+
+| Metric           | Description                                             |
+| ---------------- | ------------------------------------------------------- |
+| Accuracy         | Calculate the accuracy in percentage                    |
+| Loss             | Output the loss used for the backward pass              |
+| CPU Temperature  | Fetch the temperature of CPUs                           |
+| CPU Usage        | Fetch the CPU utilization                               |
+| CPU Memory Usage | Fetch the CPU RAM usage                                 |
+| GPU Temperature  | Fetch the GPU temperature                               |
+| Learning Rate    | Fetch the current learning rate for each optimizer step |
+| CUDA             | Fetch general CUDA metrics such as utilization          |
+
+In order to use a metric, the output of your training step has to implement the `Adaptor` trait from
+`burn-train::metric`. Here is an example for the classification output, already provided with the
+crate.
+
+```rust , ignore
+use crate::metric::{AccuracyInput, Adaptor, LossInput};
+use burn_core::tensor::backend::Backend;
+use burn_core::tensor::{Int, Tensor};
```

Comment:
Probably not meant to have these crate names (`crate` or `burn-core`) but `burn` instead.